src/app/tests/suites/certification/Test_TC_SMCO_2_5.yaml

```diff
@@ -0,0 +1,1011 @@
+      PICS: SMOKECO.S.A000b
+      command: "subscribeAttribute"
+      attribute: "SmokeSensitivityLevel"
+      minInterval: 3
+      maxInterval: 30
+      response:
+          value: 1
+          constraints:
+              type: enum8
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Smoke Sensitivity Level (High) Test
+          Event"
+      PICS: SMOKECO.S.A000b && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_HIGH
+
+    - label:
+          "TH waits for a report of SmokeSensitivityLevel attribute from DUT
+          with a timeout of 300 seconds"
+      PICS: SMOKECO.S.A000b
+      command: "waitForReport"
+      attribute: "SmokeSensitivityLevel"
+      timeout: 300
+      response:
+          value: 0
+          constraints:
+              type: enum8
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Smoke Sensitivity Level Test Event
+          Clear"
+      PICS: SMOKECO.S.A000b && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TTEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_CLEAR
+
+    - label:
+          "TH waits for a report of SmokeSensitivityLevel attribute from DUT
+          with a timeout of 300 seconds"
+      PICS: SMOKECO.S.A000b
+      command: "waitForReport"
+      attribute: "SmokeSensitivityLevel"
+      timeout: 300
+      response:
+          value: 1
+          constraints:
+              type: enum8
+
+    - label:
+          "TH sends TestEventTrigger command to General Diagnostics Cluster on
+          Endpoint 0 with EnableKey field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to
+          PIXIT.SMCO.TEST_EVENT_TRIGGER for Smoke Sensitivity Level (Low) Test
+          Event"
+      PICS: SMOKECO.S.A000b && DGGEN.S.C00.Rsp
+      cluster: "General Diagnostics"
+      endpoint: 0
+      command: "TestEventTrigger"
+      arguments:
+          values:
+              - name: "EnableKey"
+                value: TEST_EVENT_TRIGGER_KEY
+              - name: "EventTrigger"
+                value: TEST_EVENT_TRIGGER_SENSITIVITY_LEVEL_LOW
```

Comment:
The low and high sensitivity levels are optional for a server to support, but this test plan requires them.  The test plan is wrong here; let's match it for now, but make sure there is a test plan isssue (linked here) tracking fixing it.

Reply:
https://github.com/CHIP-Specifications/chip-test-plans/issues/3110